network/socket-bench-server/src/lib.rs

```diff
@@ -9,7 +9,7 @@ use futures::{
     sink::SinkExt,
     stream::{Stream, StreamExt},
 };
-use libra_crypto::test_utils::TEST_SEED;
+use libra_crypto::{test_utils::TEST_SEED, x25519, Uniform as _};
```

Comment:
why import Uniform as _, why not just Uniform, I haven't seen this style anywhere else

Reply:
in general I import traits like this, since you don't use it anywhere else you can't have a name collisions later on

Reply:
On the one hand, I agree with you. Importing traits is dumb unless you intend to explicitly use that in a box / interface. Should we make this consistent across the code base? you know let's bike shed a little ;)

Reply:
honestly I wish we would, could be enforced with a linter too (cc @sunshowers )

Reply:
idk it makes the imports kinda noisy : (
you get things like
```rust
use futures::{
    future::{Future as _, FutureExt as _},
    stream::{FusedStream as _, Stream as _, StreamExt as _},
};
```

Reply:
it'd be great if we could do 

```
use futures::{
    future::{_Future, _FutureExt},
    stream::{_FusedStream, _Stream, _StreamExt},
};
```